utils/guidelines_xslt/odd2html/specs/genericSpecFunctions.xsl

```diff
@@ -185,10 +185,11 @@
                     </xsl:for-each>
```

Comment:
Remove unused code?

Reply:
the complete code needs a cleanup, I think. Happy to do that, but not now…